test/e2e/specs/editor/various/post-editor-template-mode.spec.js

```diff
@@ -89,24 +89,20 @@ test.describe( 'Post Editor Template mode', () => {
 		await postEditorTemplateMode.disableTemplateWelcomeGuide();
 		await postEditorTemplateMode.openTemplatePopover();
 		// Change to a custom template, save and reload.
-		await page
-			.getByRole( 'menuitem', {
-				name: 'Change template',
-			} )
-			.click();
-		await page
-			.getByRole( 'option', {
-				name: 'Custom',
-			} )
-			.click();
+		await page.getByRole( 'menuitem', { name: 'Change template' } ).click();
+		await page.getByRole( 'option', { name: 'Custom' } ).click();
+		await expect(
```

Comment:
This test was previously giving a false positive. It only checks for `Single Entries` at the very end, but it failed to set the template `Custom` in the first place.